5.6/.sti/bin/assemble

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+echo "---> Installing application source ..."
+cp -Rf /tmp/src/. ./
+
+if [ -f composer.json ]; then
+  echo "Installing dependencies from composer.json... "
```

Comment:
Can you  make this message consistent with above? `" ..."`.
Also I think it would make more sense to change this to something like:

`Found 'composer.json', installing dependencies using composer.phar ...`

(so users know what is going on ;)